src/bit-systems/audio-debug-system.ts

```diff
@@ -154,6 +154,13 @@ export function audioDebugSystem(world: HubsWorld) {
   navMeshExitQuery(world).forEach(navEid => {
     removeDebugMaterial(world, navEid);
   });
+  navMeshEnterQuery(world).forEach(navEid => {
+    if (isEnabled) {
+      addDebugMaterial(world, navEid);
+    } else {
+      removeDebugMaterial(world, navEid);
```

Comment:
Does `removeDebugMaterial` really need to be called here? Does it mean there is a chance that debug material is already added?